cfme/tests/test_appliance.py

```diff
@@ -180,3 +181,28 @@ def test_keys_included(appliance, soft_assert):
         soft_assert(appliance.ssh_client.run_command(
             "test -e /var/www/miq/vmdb/certs/{}".format(k)).success,
             "{} was not included in the build".format(k))
+
+
+@pytest.mark.ignore_stream('5.9')
+def test_codename_in_log(configured_appliance):
```

Comment:
Why use `configured_appliance` here instead of `appliance`?

Reply:
I ask because its triggering the request of another sprout appliance for testing, and it doesn't appear that is necessary for this test.